### PR TITLE
Change the interval from daily to weekly in some dependabot.yml code snippets

### PR DESCRIPTION
**Pull request by mchammer01**
This PR relates to https://github.com/github/docs/pull/20734. I have swapped the `daily` value to `weekly` in a few more code snippets, as dicussed [here](https://github.com/github/docs/pull/20734#issuecomment-1255061053).

### Why:

Almost all the examples previously had `daily`.

However, we've heard from users that this often creates a little too much noise, so we changed the default in the `.github/dependabot.yaml` template file to weekly.

Since people often copy/paste examples from help docs, this updates more examples to use "weekly" as well.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Note that I left a few instances of `daily` where we explain what that cadence means.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by mchammer01** [comment]
@jeffwidman - I had time to do this work in the morning. Would you mind reviewing? Thanks 😃

**Review by newclear5555 — COMMENTED** [review]
``

**Review by jeffwidman — DISMISSED** [review]
Thanks! Really appreciate you noticing these also needed updating and then 🚢 the update as well. ❤️

File: content/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ You can also enable {% data variables.product.prodname_dependabot_version_update
...
+The example *dependabot.yml* file below configures version updates for {% data variables.product.prodname_actions %}. The `directory` must be set to `"/"` to check for workflow files in `.github/workflows`. The `schedule.interval` is set to `"weekly"`. After this file has been checked in or updated, {% data variables.product.prodname_dependabot %} checks for new versions of your actions. {% data variables.product.prodname_dependabot %} will raise pull requests for version updates for any outdated actions that it finds. After the initial version updates, {% data variables.product.prodname_dependabot %} will continue to check for outdated versions of actions once a day.
```
Comment (sophietheking) [inline content/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot.md:46]:
```suggestion
The example *dependabot.yml* file below configures version updates for {% data variables.product.prodname_actions %}. The `directory` must be set to `"/"` to check for workflow files in `.github/workflows`. The `schedule.interval` is set to `"weekly"`. After this file has been checked in or updated, {% data variables.product.prodname_dependabot %} checks for new versions of your actions. {% data variables.product.prodname_dependabot %} will raise pull requests for version updates for any outdated actions that it finds. After the initial version updates, {% data variables.product.prodname_dependabot %} will continue to check for outdated versions of actions once a week.
```

Just double checking, shouldn't this last bit say `once a week`, when the `schedule.interval` is set to `"weekly"`? 🤔

**Review by sophietheking — DISMISSED** [review]
Looks good to me! Just one small question. It makes sense to keep some of the intervals as `daily`. 👌

File: content/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot.md (hunk 1 above)
Comment (mchammer01) [reply content/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot.md:46]: Absolutely, well spotted @sophietheking 🦅 👀

**Review by sophietheking — APPROVED** [review]

**Pull request merged**